### PR TITLE
🛠 #906 Fix document title

### PR DESCRIPTION
**Pull request by stephanevieira75**
https://user-images.githubusercontent.com/36481167/123262629-5fd98180-d4f8-11eb-9c93-0e273e7d321d.mp4

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.tsx [hunk 1]
```diff
@@ -38,23 +39,29 @@ export default (props: Props) => {
...
+  const onClickChannel = () => {
+    props.id &&
+      MainViewService.select(props.id, {
+        collection: props.collection,
+        app: props.app || 'messages',
+        context: null,
+        hasTabs: props.visibility !== 'direct' && !props.app,
+      });
+
+    WindowState.setSuffix(props.name);
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.tsx:55]: Don't you think it is a duplicate ? We do it line 43 when channel load just after we click on it. For me this one should be removed.

**Review by RomaricMourgues — APPROVED** [review]
For me it's good, just a small little comment.

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.tsx (hunk 1 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.tsx:55]: If you don't set suffix here, the title won't be updated after the first load (after login for example) or reload.

https://user-images.githubusercontent.com/36481167/123264272-26a21100-d4fa-11eb-9ba1-12f5e7f4175a.mp4

Comment (RomaricMourgues) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.tsx:55]: Are you sure ? I agree that the first one (line 43) must be kept, but not the one line 55

File: twake/frontend/src/app/services/channels/channels.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import Workspaces from 'services/workspaces/workspaces.js';
 import DepreciatedCollections from 'app/services/Depreciated/Collections/Collections.js';
 import Collections from 'app/services/CollectionsReact/Collections';
 import LocalStorage from 'app/services/LocalStorage';
-import WindowService from 'services/utils/window.js';
```
Comment (chamerling) [inline twake/frontend/src/app/services/channels/channels.js:6]: Thanks for this :)

**Pull request merged**